Source/WebKit/WebProcess/GPU/media/TextTrackPrivateRemote.h

```diff
@@ -75,10 +75,10 @@ class TextTrackPrivateRemote final : public WebCore::InbandTextTrackPrivate {
     void updateConfiguration(TextTrackPrivateRemoteConfiguration&&);
 
     WebCore::TrackID id() const final { return m_id; }
-    AtomString label() const final { return AtomString { m_label.isolatedCopy() }; }
-    AtomString language() const final { return AtomString { m_language.isolatedCopy() }; }
+    String label() const final { return m_label; }
+    String language() const final { return m_language; }
     int trackIndex() const final { return m_trackIndex; }
-    AtomString inBandMetadataTrackDispatchType() const final { return AtomString { m_inBandMetadataTrackDispatchType.isolatedCopy() }; }
+    String inBandMetadataTrackDispatchType() const final { return AtomString { m_inBandMetadataTrackDispatchType.isolatedCopy() }; }
```

Comment:
Why are we doing differently than for m_label and m_language? Why do we need to call `AtomString` constructor?
Also, in some cases, we are adding isolatedCopy() and in some cases, we are removing them.
This is hard to follow.

Reply:
it's a remnant of MSE in a worker being done in the GPU process.
The MediaSource/SourceBuffer can be in a worker, but with the player in the GPU process, the TrackPrivates are created by the MediaPlayerPrivateRemote on the main thread.
So the TrackPrivate in this case is constructed on the main thread to be given back to the SourceBuffer in the worker; it needed isolatedCopy() then.

With the player in the web content process, the track is created by the SourceBufferPrivate directly and do not need this main thread jump.

I should have removed that AtomString constructor it's not needed. I'll do so.

Also, apparently StringImpl are now thread-safe so isolatedCopy will become a no-op very soon.

Reply:
I've re-visited. I've moved the responsibility of taking an isolatedCopy to the TrackPrivateClient that will be transforming the string to an AtomString. makes more sense that way